test_samples/multiple_references_to_one/Project1/obj/x64/Debug/netstandard2.0/.NETStandard,Version=v2.0.AssemblyAttributes.cs

```diff
@@ -0,0 +1,4 @@
+// <autogenerated />

```

Comment:
You should remove these files. I set gitignore to ignore this directory in attempt of keeping obj files so the roslyn would work on build server. It didn't work. Now I restore the test projects as a pipeline task and it seems to do the trick.

Reply:
Oh, thanks. I missed them in the hurry. I'll fix it tomorrow.

> Now I restore the test projects as a pipeline task...

We can add this behavior into the test execution, e. g. with `CollectionDefinition` or we extend `XunitTestFramework `.